manim/mobject/svg/svg_mobject.py

```diff
@@ -128,13 +131,87 @@ def use_to_mobjects(self, use_element):
             self.ref_to_element[ref]
         )
 
-    def attribute_to_float(self, attr):
+    def attribute_to_float(self, attr: string, default: float = 0.0) -> float:
+        # TODO: Support url(#gradient), where gradient is the id of the element to reference
+        if attr in [None, "", "none"]:
+            return float(default)
+
         stripped_attr = "".join([
             char for char in attr
             if char in string.digits + "." + "-"
         ])
         return float(stripped_attr)
 
+    """Converts a CSS color attribute value to a valid Color.
+    If it is a named CSS color, and there is a matching color in COLOR_MAP,
+    then the color defined in COLOR_MAP is applied. Note that this is likely
+    to not have the same exact value as the SVG normally would.
+    
+    Parameters
+    ----------
+    attr : :class:`string`
+        The value of the SVG 'color' attribute to convert
+        
+    Returns
+    ----------
+    :class:`Color`
+        The converted color
+        
+    Examples
+    ----------
+    Normal usage::
+        attribute_to_color(svg_element.getAttribute("color"))
+    """
+    @staticmethod
+    def attribute_to_color(attr: string) -> Color:
```

Comment:
docs go inside the function

```suggestion
    @staticmethod
    def attribute_to_color(attr: string) -> Color:
        """Converts a CSS color attribute value to a valid Color.
        If it is a named CSS color, and there is a matching color in COLOR_MAP,
        then the color defined in COLOR_MAP is applied. Note that this is likely
        to not have the same exact value as the SVG normally would.
    
        Parameters
        ----------
        attr : :class:`string`
            The value of the SVG 'color' attribute to convert
        
        Returns
        ----------
        :class:`Color`
            The converted color
        
        Examples
        ----------
        Normal usage::
            attribute_to_color(svg_element.getAttribute("color"))
        """
```